.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -44,7 +45,7 @@ stages:
           echo "PR for Image Bumping, Official Branch Cutting"
           /bin/bash vhdbuilder/scripts/automate_version_bump.sh
         env:
-          GITHUB_PAT: $(PAT-aks-node-assistant)
+          GITHUB_TOKEN: $(GITHUB_TOKEN)
```

Comment:
It seems like that you are reverting @cameronmeissner 's  change in https://github.com/Azure/AgentBaker/pull/5339/files

Reply:
We moved away from the PAT. We now have a github app